CDCgov/prime-reportstream
# Added constants to the FHIR Path tool

This PR adds the ability to add constants to the FHIR Path tool.  For example:
```
./prime fhirpath -i src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_2022051 -c patientState=Bundle.entry.resource.ofType(Patient).address.state -c patientZip=Bundle.entry.resource.ofType(Patient).address.postalCode -c patientCountry=Bundle.entry.resource.ofType(Patient).address.country
```

Test Steps:
1. Run `./gradlew fatjar`
2. Run the prime command above in the description.
3. Type in `%patientZip` and press Enter. Verify the zip code value is provided.

## Changes
- Added constants to the FHIR path tool

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
